java/vector/src/main/java/org/apache/arrow/vector/BitVectorHelper.java

```diff
@@ -73,6 +87,28 @@ public static void setBit(ArrowBuf validityBuffer, int index) {
     validityBuffer.setByte(byteIndex, currentByte);
   }
 
+  /**
+   * Set the bit at provided index to 1.
+   *
+   * @param validityBuffer validity buffer of the vector
+   * @param index index to be set
+   */
+  public static void setBit(ArrowBuf validityBuffer, long index) {
+    // it can be observed that some logic is duplicate of the logic in setValidityBit.
```

Comment:
I think we can replace the old implementation 
```
void setBit(ArrowBuf validityBuffer, int index)
```
with this one, as now ArrowBuf is based on 64-bit index.

Reply:
done

Reply:
done